data_model/README.md

```diff
@@ -2,30 +2,115 @@
 orphan: true
 ---
 
-## Contents
+# Data model XML files
 
 This folder contains a machine-readable representation of matter clusters.
 
-The XML files inside `clusters` are generated by a `scraper` script out of the
-original specification `AsciiDoc` files.
+These files are currently used for certification testing and documentation. The
+data model files represent the official specification data model at a certain
+revision. The files in these directories are maintained by the data model tiger
+team (DMTT) and are updated when new official specifications or specification
+ballots are released.
 
-## How to update
+The data model files in this directory are not used with zap or for SDK codegen
+and should not be updated manually by cluster or device type implementers.
 
-The matter specification is not currently public. As such, as script exists to
-update the spec XML files, however this is not done automatically.
+## Updating the data model files
 
-You will require access to the following tools locally:
+### Updating current spec revision directories
 
--   `scraper`. A binary copy generally available
-    [here](https://github.com/csa-data-model/projects/tree/main/DM-Editor/bin/scrape)
--   Specification repository checkout from
-    https://github.com/CHIP-Specifications/connectedhomeip-spec
+If the files exist in the data_model directory already and you are trying to
+update either the revision or the alchemy release, use
+scripts/spec_xml/generate_spec_xml.py. More information on the use of the script
+can be found by using the --help flag for that script.
```

Comment:
Is there a `typical command to run` ? The previous shell example of `generate_spec_xml` seemed really useful to see it at a glance. `Use --help` is more painful.

Reply:
added.